ydb/docs/en/core/concepts/federated_query/greenplum.md

```diff
@@ -0,0 +1,76 @@
+# Working with Greenplum databases
+
+This section provides basic information on working with external [Greenplum](https://greenplum.org) databases. Since Greenplum is based on PostgreSQL technology, some of their points are the same, so some links may lead to PostgreSQL documentation.
```

Comment:
```suggestion
This section provides basic information on working with external [Greenplum](https://greenplum.org) databases. Since Greenplum is based on [PostgreSQL](postgresql.md), integrations with them are similar, and some links below may lead to PostgreSQL documentation.
```